src/extensions/bullet/bulletext_hooks.cpp

```diff
@@ -736,6 +739,65 @@ DECLARE_PATCH(_BulletClass_AI_Intercept)
     JMP_REG(ecx, 0x00444707); // Jump to IsActive check
 }
 
+
+/**
+ *  Halves the distance from the explosion to the unit for units in air.
```

Comment:
Is this necessary? This will have an impact on gameplay that we could likely achieve by just increasing Spread= on AA weapons.

Reply:
This is already performed by the game for Aircraft. I just added flying Vehicles to it, which mirrors YR.

Reply:
You should include that information in the comment there so it doesn't look like a random gameplay change.